release/src/main/python-release/run_release_candidate_python_mobile_gaming.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+#  This file will verify Apache/Beam release candidate python by following steps:
+#
+#  1. Create a new virtualenv and install the SDK
+#  2. Run UserScore examples with DirectRunner
+#  3. Run UserScore examples with DataflowRunner
+#  4. Run HourlyTeamScore on DirectRunner
+#  5. Run HourlyTeamScore on DataflowRunner
+#
+
+set -e
+set -v
+
+source release/src/main/python-release/python_release_automation_utils.sh
+
+# Assign default values
+BEAM_PYTHON_SDK=$BEAM_PYTHON_SDK_ZIP
+
+
+#######################################
+# Remove temp directory when complete.
+# Globals:
+#   TMPDIR
+# Arguments:
+#   None
+#######################################
+function complete() {
+  print_separator "Validation $1"
+  rm -rf $TMPDIR
+}
+
+
+#######################################
+# Download files from RC staging location, install python sdk
+# Globals:
+#   BEAM_PYTHON_SDK
+# Arguments:
+#   None
+#######################################
+function install_sdk() {
+  print_separator "Creating new virtualenv and installing the SDK"
+  virtualenv temp_virtualenv
+  . temp_virtualenv/bin/activate
+  gcloud_version=$(gcloud --version | head -1 | awk '{print $4}')
+  if [[ "$gcloud_version" < "189" ]]; then
+    update_gcloud
+  fi
+  pip install google-compute-engine
+  pip install $BEAM_PYTHON_SDK[gcp]
+}
+
+
+#######################################
+# Run UserScore with DirectRunner
+# Globals:
+#   USERSCORE_OUTPUT_PREFIX, DATASET, BUCKET_NAME
+# Arguments:
+#   None
+#######################################
+function verify_userscore_direct() {
+  print_separator "Running userscore example with DirectRunner"
+  output_file_name="$USERSCORE_OUTPUT_PREFIX-direct-runner.txt"
+  python -m apache_beam.examples.complete.game.user_score \
+    --output=$output_file_name \
+    --project=$PROJECT_ID \
+    --dataset=$DATASET \
+    --input=gs://$BUCKET_NAME/5000_gaming_data.csv
+
+  verify_user_score "direct"
```

Comment:
What does this line do? (Similar comment for a few other calls inside functions to the wrapping function.)

Reply:
These lines calls the helper functions in the python_release_automation_utils.sh to verify results of example pipelines (e.g verifying whether output files were created properly, checking if expected keywords show in the results, etc).

Reply:
Thanks. I'll pay attention to it.